src/deluge/gui/ui/keyboard/chords.h

```diff
@@ -92,8 +120,21 @@ extern const Chord kM11;
 extern const Chord kMinor11;
 extern const Chord k13;
 extern const Chord kM13;
+extern const Chord kM13Sharp11;
 extern const Chord kMinor13;
 
+extern const std::vector<Chord> majorChords;
```

Comment:
Small dynamic allocations will lead to fragmentation - can you use a static vector? 

In most programs that's handled by getting fresh pages of memory from the OS, but as we are the OS and we don't have defragmentation implemented our strategy is to avoid fragmenting in the first place

Reply:
Yeah I can do that tomorrow morning!

Reply:
Fixed!